### PR TITLE
Feature/search bar ui

### LINKED ISSUE #50: Reusable Search Bar UI
Create a reusable component we'll call "Search Bar". It should:

* Have a magnifying glass icon
* Have a configurable title
* Be tappable
* Bring up the keyboard (e.g. have a text field embedded)

**Design**
Check on the screenshot below
[Link to Design](https://www.figma.com/file/4cV84H1jyMo0rJEAl3HNnJ/FightPandemics-landing-page?node-id=6638%3A29348)

**EXAMPLES**:

Screen Shot 2020-05-13 at 9 45 40 PM

Screen Shot 2020-05-13 at 9 45 51 PM

### PR DESCRIPTION
**Pull request by ethanswift**
**What this PR does**:
Creates a reusable search bar UI which takes string as a parameter and make a search bar with the specified placeholder.
Screen Shot 2020-05-14 at 10 30 41 PM

### Checklist

[x] Compiler warnings resolved
[x] Linter warnings resolved
[x] User-facing strings in Localizable.strings file
[x] Unused code, print statements, and comments removed

**Which issue(s) this PR fixes**:

Resolves #

**Special notes for reviewers**:
Feed view is assigned as home view controller for testing purposes. 
**Additional comments**:

### CODE REVIEW

**Review by haidalinda — COMMENTED** [review]
we're going to need to resolve the merge conflicts in the parent branch before merging both of these - i can handle that

**Comment by haidalinda** [comment]
generally regarding merge conflicts @ethanswift - before you begin new work, make sure you're pulling down the latest `develop` and branching off of that. i know the codebase and project file are changing quickly at the moment, so its harder to keep in-sync. but over time we'll want to minimize the time spent resolving conflicts

File: FightPandemics/Features/Feed/FeedViewController.swift [hunk 1]
```diff
@@ -52,6 +53,7 @@ class FeedViewController: UIViewController {
         self.avatarView.setUp(user: mockUser, avatar: self.avatar)
         self.createPostBody.setUp()
         self.createPostFooter.setUp()
+        self.searchBar.setUp(placeHolder: "Search any thing")
```
Comment (stavares843) [inline FightPandemics/Features/Feed/FeedViewController.swift:56]: Search any things

File: FightPandemics/Features/Feed/Search Bar/SearchBar.swift [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+//  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+//  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+//  THE SOFTWARE.
+
+import UIKit
+
+class SearchBar: UIView {
+    private var searchBar = UITextField()
+    private var filterBtn = UIButton()
+    func setUp (placeHolder: String) {
```
Comment (stavares843) [inline FightPandemics/Features/Feed/Search Bar/SearchBar.swift:32]: maybe placeholder

File: FightPandemics/Features/Feed/Search Bar/SearchBar.swift [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+        lImgView.frame = CGRect(x: 19, y: 13, width: 25, height: 25)
+        lView.addSubview(lImgView)
+        searchBar.leftView = lView
+        searchBar.leftViewMode = .always
+        searchBar.frame.size.height = 45
+        searchBar.layer.masksToBounds = true
+        searchBar.layer.cornerRadius = searchBar.frame.size.height / 2
+        searchBar.backgroundColor = UIColor(hexString: "#F2F2F2")
+        self.addSubview(searchBar)
+        filterBtn.setBackgroundImage(#imageLiteral(resourceName: "Action 19"), for: .normal)
```
Comment (stavares843) [inline FightPandemics/Features/Feed/Search Bar/SearchBar.swift:45]: do we have any convention for assets naming? @hkellaway

**Pull request merged**